src/screens/Asset/ReceiveModal.js

```diff
@@ -18,31 +18,52 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 import * as React from 'react';
+import { connect } from 'react-redux';
 import { View, Image, Dimensions, Share, Clipboard } from 'react-native';
 import { SafeAreaView } from 'react-navigation';
-import { BaseText } from 'components/Typography';
-import { spacing, fontStyles, fontSizes } from 'utils/variables';
 import styled from 'styled-components/native';
+import { withTheme } from 'styled-components';
+
+// components
+import { BaseText, MediumText } from 'components/Typography';
 import SlideModal from 'components/Modals/SlideModal';
 import Button from 'components/Button';
 import WarningBanner from 'components/WarningBanner';
 import QRCodeWithTheme from 'components/QRCode';
 import { LabelBadge } from 'components/LabelBadge';
 import Toast from 'components/Toast';
+import ProfileImage from 'components/ProfileImage';
+
+// utils
+import { spacing, fontStyles, fontSizes } from 'utils/variables';
+import { getEnsName, getAccountTypeByAddress } from 'utils/accounts';
+import { getThemeColors, themedColors } from 'utils/themes';
+
+// models and types
+import type { Accounts } from 'models/Account';
+import type { RootReducerState } from 'reducers/rootReducer';
+import type { Theme } from 'models/Theme';
+import type { User } from 'models/User';
+
+// constants
+import { ACCOUNT_TYPES } from 'constants/accountsConstants';
 
 const ContentWrapper = styled(SafeAreaView)`
-  padding: 0 ${spacing.layoutSides}px 60px;
+  padding: 0 ${spacing.layoutSides}px 50px;
```

Comment:
Seems like wrapper should have bigger paddings on the sides: https://app.zeplin.io/project/5db9b5d14c30ae56faaa3030/screen/5efdff1fae9030a1cd837873